solidity/contracts/isms/multisig/AbstractMessageIdMultisigIsm.sol

```diff
@@ -9,9 +9,14 @@ import {Message} from "../../libs/Message.sol";
 import {CheckpointLib} from "../../libs/CheckpointLib.sol";
 
 /**
- * @title AbstractMessageIdMultisigIsm
- * @notice Provides abstract logic for verifying signatures on a message ID.
- * @dev Implement and use if you want fastest and cheapest security.
+ * @title `AbstractMessageIdMultisigIsm` — multi-sig ISM for the censorship-friendly validators.
+ * @notice This ISM minimizes gas/performance overhead of the checkpoints verification by compromising on the censorship resistance.
+ * For censorship resistance consider using `AbstractMerkleRootMultisigIsm`.
+ * If the validators (`validatorsAndThreshold`) skip messages by not sign checkpoints for them,
+ * the relayers will not be able to aggregate a quorum of signatures sufficient to deliver these messages via this ISM.
+ * Integrations are free to choose the trade-off between the censorship resistance and the gas/processing overhead.
```

Comment:
We are actually planning to configure a 1/2 aggregation ISM with both of these paths available

Reply:
Ok, nice! I think you can extend the doc on the `AggregationIsm` after releasing this.